source/material_model/melt_simple.cc

```diff
@@ -96,6 +97,7 @@ namespace aspect
                     const double maximum_melt_fraction,
                     const NonlinearDependence::Dependence dependence) const
     {
+
       double entropy_gradient = 0.0;
 
       // calculate latent heat of melting
```

Comment:
I do not understand how this still works, currently, since you're not reading in these parameters anymore. 
I think you need to move all of this into your melting model as well. 

This also means that apparently, we do not have a test for the latent heat of melting in the melt simple model at the moment, so it would be great if you could add one!